zqd/handlers_test.go

```diff
@@ -634,7 +634,7 @@ func TestDeleteDuringPcapPost(t *testing.T) {
 	err = client.SpaceDelete(context.Background(), sp.ID)
 	require.NoError(t, err)
 
-	require.Error(t, <-pcapPostErr, "context canceled")
+	require.Regexp(t, "context canceled", (<-pcapPostErr).Error())
```

Comment:
More specific and clearer to do this:
```suggestion
	require.EqualError(t, <-pcapPostErr, "context canceled")
```